syntax/common.vim

```diff
@@ -41,6 +41,9 @@ runtime syntax/ts-common/cluster.vim
 runtime syntax/ts-common/function.vim
 runtime syntax/ts-common/decorator.vim
 
+hi def link typescriptIdentifierName        Identifier
+hi def link typescriptProp                  Identifier
```

Comment:
should Prop be linked to Identifier? It will make both option and field in expression like`option.field` highlighted as identifiers.

Also, can you paste a screenshot for the change? thanks

Reply:
Vim only has `Identifier`, so it doesn't distinguish between variables and fields. Currently, `option` and `field` already look the same since both are unlinked.